src/core/Akka.Remote.Tests/Serialization/BugFix5062Spec.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Remote.Configuration;
+using Akka.Serialization;
+using Akka.TestKit;
+using FluentAssertions;
+using Google.Protobuf;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Remote.Tests.Serialization
+{
+    public class BugFix5062Spec: AkkaSpec
+    {
+        private static Config DDataConfig => ConfigurationFactory.ParseString($@"
+            akka.actor {{
+                serializers {{
+                    dummyWithManifest = ""{typeof(DummySerializerWithStringManifest).AssemblyQualifiedName}""
+                }}
+                serialization-bindings {{
+                    ""{typeof(SomeMessage).AssemblyQualifiedName}"" = dummyWithManifest
+                }}
+                serialization-identifiers {{
+	                ""{typeof(DummySerializerWithStringManifest).AssemblyQualifiedName}"" = 13
+                }}
+            }}")
+            .WithFallback(RemoteConfigFactory.Default());
+
+        public BugFix5062Spec(ITestOutputHelper output) : base(output, DDataConfig)
+        { }
+
+        [Fact]
+        public void Failed_serialization_should_give_proper_exception_message()
+        {
+            var message = new ActorSelectionMessage(
+                new SomeMessage(), 
+                new SelectionPathElement[] { new SelectChildName("dummy") }, 
+                true);
+
+            var node1 = new Address("akka.tcp", "Sys", "localhost", 2551);
+            var serialized = MessageSerializer.Serialize((ExtendedActorSystem)Sys, node1, message);
+
+            var o = new object();
+            var ex = o.Invoking(s => MessageSerializer.Deserialize((ExtendedActorSystem)Sys, serialized)).Should()
+                .Throw<SerializationException>()
+                .WithMessage("Failed to deserialize object with serialization id [6] (manifest []).")
+                //.WithInnerExceptionExactly<SerializationException>()
+                //.WithMessage("Failed to deserialize object with serialization id [11] (manifest [E]).")
+                .WithInnerExceptionExactly<SerializationException>()
+                .WithMessage("Failed to deserialize object with serialization id [13] (manifest [SM]).");
```

Comment:
Your use case and reproduction here is wrong - need to have a scenario where `SomeMessage` isn't registered at all, not that it fails during deserialization.

The goal of this bug is to unpack the inner exception and throw _that_ - so the user gets a clear error message: that the payload carried by an `ActorSelection` couldn't be found because of a missing manifest, not that the `ActorSelection` _itself_ couldn't be deserialized.